ovos_utils/sound/__init__.py

```diff
@@ -32,110 +33,86 @@ def _get_pulse_environment(config):
     else:
         return os.environ
 
-
-def play_acknowledge_sound():
-    """Acknowledge a successful request.
-
-    This method plays a sound to acknowledge a request that does not
-    require a verbal response. This is intended to provide simple feedback
-    to the user that their request was handled successfully.
-    """
+def _play_specific_sound(sound_name, bus=None):
```

Comment:
possibly out of scope for this PR, but what happens if `bus` is None? it appears that we issue a deprecation warning in `play_sound()` and then... nothing?

Reply:
this is a step in supporting distributed systems, such as ovos-docker, but it's only one of many steps, currently we officially dont support running the ovos stack a distributed way so the old behaviour is valid. finding a bus would introduce unacceptable latency in sounds supposed to be instant

Reply:
if bus is None it complains and falls through to the existing code which will run an external player and do what it used to do.